api/python/quilt3/session.py

```diff
@@ -237,6 +237,16 @@ def _refresh_credentials():
     return result
 
 
+def logged_in():
+    """
+    Return registry URL if Quilt client is authenticated. Otherwise
+    return `None`.
+    """
+    url = get_registry_url()
+    if url in _load_auth():
+        return url
```

Comment:
This seems to return the registry_url. The more meaningful URL is the catalog URL, which is the URL the user specifies in `quilt3.login`. The username might also be as useful here if not more useful.

Reply:
> This seems to return the registry_url. The more meaningful URL is the catalog URL, which is the URL the user specifies in quilt3.login

That's the same URL that is used in `quilt3.login()`:
https://github.com/quiltdata/quilt/blob/43abfd13ffa2411a553550bb7181505c6dd66481/api/python/quilt3/session.py#L166-L191

> The username might also be as useful here if not more useful.

If I don't miss something we can't return username, as we store only refresh/access tokens.

Reply:
We should return `navigator_url` from `get_from_config`.